charts/nginx-ingress-control-plane.yaml

```diff
@@ -0,0 +1,42 @@
+controller:
+  image:
+    repository: '{%- endraw -%}{{ build_image_name("nginx-ingress-controller", False) }}{%- raw -%}'
+
+  defaultBackendService: 'metalk8s-ingress/nginx-ingress-default-backend'
```

Comment:
we may need a good 404 page with the same theme as the UI

Reply:
Indeed, could even redirect to MetalK8s UI 404 page.

Reply:
Good idea, but I think that's out of scope for this PR.